### ISSUE TITLE
Exiting UnrealModLoader also shuts down the game without a warning. [Star wars: The fallen order]

### ISSUE THREAD

**Issue by Chummpp** [RussellJerome/UnrealModLoader#39]
Exiting UML also shuts down the game.

**Comment by Fivebuss** [comment]
This is how DLL attachments work often, normal behavior

**Comment by Cracko298** [comment]
Yeah, some games like these have detection for DLL injections too. (Especially AAA games)

**Comment by Fivebuss** [comment]
> Yeah, some games like these have detection for DLL injections too. (Especially AAA games)

If it detects DLL injection it would crash before you close the program no?
